### DISCUSSION TITLE
国内访问

### DISCUSSION THREAD

**Discussion by magisk317** [spencerwooo/onedrive-vercel-index#816] [Q&A] [answered]
翻遍了所有的讨论，发现竟然没人问这个问题(可能是我太傻了😵)，我自己翻出去没问题，但是分享给别人怎么弄呢？
没怎么搞过这种，我觉得第一遍能不报错搭起来已经很不容易了😂

**Comment by zkeq** [comment, marked as answer]
`vercel.app` 国内被🧱，绑定自己域名即可

**Reply by magisk317** [reply]
嗯，后来我试了下，确实可以，感谢回答

另外，国内有没有类似平台啊，不用绑定域名直接用的？

**Reply by wdw1007** [reply]
> 另外，国内有没有类似平台啊，不用绑定域名直接用的？

有的
netlify

**Reply by magisk317** [reply]
> > 另外，国内有没有类似平台啊，不用绑定域名直接用的？
> 
> 有的 netlify

好的，多谢，我试试

**Reply by wdw1007** [reply]
哦抱歉刚发现你说的是国内
这个我不太清楚还请谅解（
